filament/include/filament/IndirectLight.h

```diff
@@ -143,6 +173,38 @@ class UTILS_PUBLIC IndirectLight : public FilamentAPI {
          */
         Builder& irradiance(uint8_t bands, math::float3 const* sh) noexcept;
 
+        /**
+         * Sets the irradiance from the radiance expressed as Spherical Harmonics.
```

Comment:
Do you mean `radiance` here?

Reply:
Well, no, it's setting the irradiance from the radiance. I guess both irradiance() and radiance() methods can be used to specify the irradiance.